0.Data_Preprocessing/scripts/0.generate_toy_data.py

```diff
@@ -0,0 +1,70 @@
+# ---
+# jupyter:
+#   jupytext:
```

Comment:
If it hasn't already been done, consider adding `jupytext` as a development dependency for this project.